lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -1045,6 +1045,7 @@ def _ng_qer_update(
                 )
             if qos_enforce_rule.ipv6_addr:
                 ipv6 = convert_ipv6_bytes_to_ip_proto(qos_enforce_rule.ipv6_addr)
+                self._update_ipv6_prefix_store(qos_enforce_rule.ipv6_addr)
```

Comment:
just curious, why is the store update not needed in the `PdrState.Value('IDLE')` case?

Reply:
Yes, it's required but for idle case fully not supported or verified. Once will start the work for Idle mode for IPv6 then will add it.